apps/buildog/src/app/account/settings/profile/page.tsx

```diff
@@ -77,7 +105,13 @@ export default function ProfilePage() {
                   <FormItem>
                     <FormLabel>First Name</FormLabel>
                     <FormControl>
-                      <Input className="w-96" type="text" placeholder="John" {...field} />
+                      <Input
+                        className="w-96"
+                        type="text"
+                        placeholder={userCredentials.first_name}
+                        disabled={true}
+                        {...field}
+                      />
```

Comment:
If the inputs are disabled, you can remove the used useForm hooks and Form components.